THE-app/components/BookCard.tsx

```diff
@@ -11,23 +12,39 @@ interface BookCardProps {
 const windowWidth = Dimensions.get('window').width;
 
 const BookCard: React.FC<BookCardProps> = ({ book }) => {
+
   return (
-    <View style={styles.card}>
-      {book.url ? (
-        <Image source={{ uri: book.url }} style={styles.image}/>
-      ) : (
-        <View style={styles.imagePlaceholder}>
-          <Text>No Image</Text>
-        </View>
-      )}
-      <BlurView style={styles.overlay} tint="dark" intensity={25} experimentalBlurMethod='dimezisBlurView'>
-        <Text style={styles.title}>{book.displayTitle}</Text>
-      </BlurView>
-    </View>
+    <Link href={`/chapterScreen?bookId=${book.id}`} asChild>
```

Comment:
It's better to use the object version of the `href` prop, like `{ pathname: "/chapterScreen", params: { bookId: book.id } }` (one reason is that it doesn't require you to properly escape the URL parameters).